From 014be5698ed0645f1d5263b354c27fc3ba77b281 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Sat, 14 Mar 2026 18:01:51 +0100
Subject: [PATCH 03592/15333] MAINTAINERS: Update remoteproc repo url for
 hwspinlock

Since 2021, the remoteproc repo is not hosted anymore in Bjorn's
personal namespace, but commit cc73f503f7ec ("MAINTAINERS: Update
remoteproc repo url") only updated the url for remoteproc and
rpmsg in MAINTAINERS file.
The old repository is still accessible, but it's not updated since
2021 and is not anymore listed in https://git.kernel.org/ .

Update the url for hwspinlock too.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Link: https://lore.kernel.org/r/20260314170151.18319-1-antonio.borneo@foss.st.com
Signed-off-by: Bjorn Andersson <andersson@kernel.org>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 15011f5752a9..477be16693db 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11430,7 +11430,7 @@ M:	Bjorn Andersson <andersson@kernel.org>
 R:	Baolin Wang <baolin.wang7@gmail.com>
 L:	linux-remoteproc@vger.kernel.org
 S:	Maintained
-T:	git git://git.kernel.org/pub/scm/linux/kernel/git/andersson/remoteproc.git hwspinlock-next
+T:	git https://git.kernel.org/pub/scm/linux/kernel/git/remoteproc/linux.git hwspinlock-next
 F:	Documentation/devicetree/bindings/hwlock/
 F:	Documentation/locking/hwspinlock.rst
 F:	drivers/hwspinlock/
-- 
2.55.0

